protoc-gen-openapiv2/options/openapiv2.proto

```diff
@@ -97,6 +97,9 @@ message Swagger {
   repeated Tag tags = 13;
   // Additional external documentation.
   ExternalDocumentation external_docs = 14;
+  // Custom properties that start with "x-" such as "x-foo" used to describe
+  // extra functionality that is not covered by the standard OpenAPI Specification.
+  // See: https://swagger.io/docs/specification/openapi-extensions/
```

Comment:
This links to the OpenAPIv3 spec page, please update the link to https://swagger.io/docs/specification/2-0/swagger-extensions/.